canopen/pdo/__init__.py

```diff
@@ -77,3 +85,7 @@ def stop(self):
                 pdo.stop()
         else:
             raise TypeError('The node type does not support this function.')
+
+
+# Compatibility
+Variable = PdoVariable
```

Comment:
I wonder if `Variable` should be listed in `__all__`. In principle `__all__` contains the entire public API of the module. However, it's not very common to list objects defined in the file in `__all__` is it?

Reply:
I don't know what's common practice with regard to that `__all__` attribute.  But my understanding is that it controls what `from ... import *` does.  In that case, `Variable` should not be imported, as it is a deprecated alias only kept for compatibility.  But if someone references it directly, that still works.

Reply:
Svein:
> However, it's not very common to list objects defined in the file in `__all__` is it?

That's of course possible; for example, a sentinel or a singleton could also be a part of a public API.

André:
> [...] my understanding is that it controls what `from ... import * does`.
> [...] But if someone references it directly, that still works.

Both are correct. `from canopen.pdo import Variable` still works.